backend/EasyMeets.Emailer/EasyMeets.Emailer.WebAPI/Services/EmailService.cs

```diff
@@ -1,44 +1,40 @@
 ﻿using EasyMeets.Emailer.WebAPI.Services.Interfaces;
 using sib_api_v3_sdk.Api;
-using sib_api_v3_sdk.Client;
 using sib_api_v3_sdk.Model;
 
 namespace EasyMeets.Emailer.WebAPI.Services
 {
     public class EmailService : IEmailService
     {
-        private readonly string _apiKey;
         private readonly string _senderEmail;
         private readonly string _senderName;
         private readonly ILogger<EmailService> _logger;
         private readonly TransactionalEmailsApi _api;
 
         public EmailService(IConfiguration configuration, ILogger<EmailService> logger)
         {
-            _apiKey = configuration["Sendinblue:ApiKey"];
             _senderEmail = configuration["Sender:Email"];
             _senderName = configuration["Sender:Name"];
             _logger = logger;
             _api = new TransactionalEmailsApi();
         }
 
-        public async Task<string> SendEmail(string recipient, string body, string subject)
+        public async Task<string> SendEmailAsync(string recipient, string body, string subject)
         {
-            Configuration.Default.ApiKey.Add("api-key", _apiKey);
-            var Email = new SendSmtpEmailSender(_senderName, _senderEmail);
+            var email = new SendSmtpEmailSender(_senderName, _senderEmail);
             var smtpEmailTo = new SendSmtpEmailTo(recipient);
-            List<SendSmtpEmailTo> To = new() { smtpEmailTo};
+            var to = new List<SendSmtpEmailTo> { smtpEmailTo };
             try
             {
-                var sendSmtpEmail = new SendSmtpEmail(Email, To, null, null, null, body, subject);
-                CreateSmtpEmail result = await _api.SendTransacEmailAsync(sendSmtpEmail);
+                var sendSmtpEmail = new SendSmtpEmail(email, to, null, null, null, body, subject);
+                var result = await _api.SendTransacEmailAsync(sendSmtpEmail);
 
                 return result.MessageId;
             }
             catch (Exception e)
             {
                 _logger.LogError(e, "Email wasn't sent");
-                return string.Empty;
+                throw;
```

Comment:
why just "throw" without "throw ...." and some exception, then we will can know from exception message what went wrong

Reply:
Here, "throw" is used to re-throw an existing exception without changing the stack trace